<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestCreator</class>
 <widget class="QDialog" name="TestCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание теста</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_test_name">
      <property name="placeholderText">
       <string>Название теста</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_questions"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_add_question">
        <property name="text">
         <string>Добавить вопрос</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove_question">
        <property name="text">
         <string>Удалить вопрос</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save_test">
      <property name="text">
       <string>Сохранить тест</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
